### PR TITLE
Updating the deploy-apps script

### LINKED ISSUE #26: Make deploy-apps to consume enviroments
We have too many apps and environments right now, and they are all CPU or memory intensive. This made them require too many resources to run. We should make the `deploy-apps` scripts consume a list of apps to deploy.

### PR DESCRIPTION
**Pull request by QuinceyJames**
# Problem
- Currently, the deploy-apps.sh script has no way of selectively choosing which apps to deploy. This means that users will have to wait a while for all the apps to build. Further, running all of these apps is resource-intensive. 

# Changes
- Edited the deploy-apps.sh script to allow the user to choose which apps to deploy
- updated the readme to include an example of how to use the script

# Issues
- Fixes #26

### CODE REVIEW

**Comment by QuinceyJames** [comment]
Hi there! Thanks for creating such a cool project. This is my first time contributing to something open source so please forgive any blatant mistakes I made 😄

**Review by kakkoyun — APPROVED** [review]
LGTM.
It is really fancy! Thanks for the contribution.
I have just one nit suggestion.

@maxbrunet What do you think?

**Review by maxbrunet — CHANGES_REQUESTED** [review]
Fancy indeed! :sparkles:

Just a few comments to ensure we do not fall into odd states

I am going to enable some linters on the repo and once these pass on the PR, we can get it merged

File: deploy-apps.sh
Comment (QuinceyJames) [reply deploy-apps.sh]: EOF Blank line added

File: deploy-apps.sh
Comment (QuinceyJames) [reply deploy-apps.sh]: - Shebang updated
- Strict mode added

File: deploy-apps.sh
Comment (QuinceyJames) [reply deploy-apps.sh]: - print statement fixed

File: deploy-apps.sh
Comment (QuinceyJames) [reply deploy-apps.sh:38]: - local variables used (but please double check I didn't miss any)

**Comment by QuinceyJames** [comment]
Thanks so much for your feedback! Never heard of strict mode before -- it caught some strange behavior in ```findDeploymentIndexFromInteger```.

Btw I'm not sure if I am meant to mark the conversations as resolved or if I leave that you.

Also, should I pull in the linter changes from your repo into my fork and run it? Or will Github warn me automatically? 

Thanks again!

**Comment by maxbrunet** [comment]
> Thanks so much for your feedback! Never heard of strict mode before -- it caught some strange behavior in `findDeploymentIndexFromInteger`.

You're welcome!

> Btw I'm not sure if I am meant to mark the conversations as resolved or if I leave that you.

Yes, I think you can resolve them

> Also, should I pull in the linter changes from your repo into my fork and run it? Or will Github warn me automatically?

If you want to get a head start, you can run `shellcheck deploy-apps.sh` locally: https://github.com/koalaman/shellcheck#installing
It's mostly this one that may give you trouble.

**Comment by kakkoyun** [comment]
@QuinceyJames Could you please rebase? Now we have some CI checks, I'm good to merge when the CI is green https://github.com/parca-dev/parca-demo/pull/31/files

**Review by maxbrunet — APPROVED** [review]
Great work! Thank you!

**Pull request merged**
